locale/foreman_leapp.pot

```diff
@@ -166,19 +166,19 @@ msgid "Inhibitor?"
 msgstr ""
 
 #: action_names.rb:2
-msgid "Remote action:"
+msgid "Action with sub plans"
```

Comment:
This is just shuffling lines around. https://github.com/theforeman/foreman-tasks/pull/698 should make it stable in the future.